bundle/executor.go

```diff
@@ -63,14 +74,17 @@ type executor struct {
 	lastStatus           StatusMessage
 	statusChan           chan StatusMessage
 	mutex                sync.Mutex
+	stateManager         StateManager
 }
 
 // NewExecutor - Creates a new Executor for running an APB.
-func NewExecutor() Executor {
+func NewExecutor(k8sClient k8.Interface, stateNS string) Executor {
```

Comment:
not sure if this is the right approach. Feedback welcome

Reply:
I would prefer to use the clients package to get the k8s client that way. That is how most things are behaving and I have examples of how to test that.

Reply:
Is ```stateNS``` different than ```bundle.serviceInstance.Context.Namespace``` https://github.com/automationbroker/bundle-lib/blob/master/bundle/types.go#L140 ?